static/operator.yaml

```diff
@@ -142,18 +142,32 @@ rules:
   - get
   - create
 - apiGroups:
-  - apps
-  resourceNames:
-  - istio-operator
+  - policy
   resources:
-  - deployments/finalizers
+  - poddisruptionbudgets
   verbs:
-  - update
+  - '*'
 - apiGroups:
-  - install.istio.io
+  - rbac.authorization.k8s.io
   resources:
+  - clusterrolebindings
+  - clusterroles
+  - roles
+  - rolebindings
+  verbs:
   - '*'
-  - istiocontrolplanes
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - endpoints
+  - events
+  - namespaces
+  - pods
+  - persistentvolumeclaims
+  - secrets
+  - services
+  - serviceaccounts
   verbs:
   - '*'
 ---
```

Comment:
might as well terminate every yaml in the istio/operator/deploy directory so this is consistently rendered.